package.json

```diff
@@ -110,6 +110,7 @@
     "karma-ng-html2js-preprocessor": "^0.1.2",
     "karma-phantomjs-launcher": "~0.2.0",
     "load-grunt-tasks": "^3.2.0",
+    "protractor": "git://github.com/angular/protractor.git#master",
```

Comment:
for some reason, this isn't doing the trick with node v.10 even though this PR was thought to fix the issue.
https://github.com/angular/protractor/pull/2591/files
https://travis-ci.org/angular/protractor/jobs/84351567

Anyone have thoughts on this?